provider-tutorials/provider-cli.md

```diff
@@ -221,6 +221,162 @@ For `--network`you have two options, either `mainnet` or `rinkeby`. For `--to-ad
 
 **Note that if you decided to use an external wallet during your setup process, you can connect to ZkSync's wallet at** [**https://wallet.zksync.io/**](https://wallet.zksync.io/) **and exit that way. In the scenario that a different payment driver is being used, you will need to use the relevant available options to connect and access your tokens.**
 
+## Outbound
```

Comment:
```suggestion
## Outbound Network
```

Reply:
What is more important is that is is incorrectly placed, and its content is too broad:
1. This page mainly describes `golemsp`, which goal is to hide `ya-provider` CLI complexity from user.
Because of `keystore`  and `whitelist` cmd I created paragraph `Advanced Settings` which describes some commands of `ya-provider`, so it should be moved there, or maybe description of `ya-provider` CLI commands should get completely new page?
2. This page describes some details of `ya-provider` / `golemsp` features only in relation to CLI commands. Meanwhile the new chapter describes some `Outbound`. Instead if anything it should describe `Rule` CLI command. As a part of it  it should describe that it allows to `set` and `list` rules, and then it could mention that one of the rules is about `Outbound Network`.
3. Maybe `Outbound Network` description should be a part of https://github.com/golemfactory/yagna-docs/blob/develop/requestor-tutorials/vm-runtime/computation-payload-manifest.md?plain=1#L129 or be a dedicated page and links from `computation-payload-manifest` and from this page (`ya-provider` CLI docs) would lead there?

Reply:
I agree. I will try to find better place for it